src/apps/shutdown/shutdown.c

```diff
@@ -0,0 +1,17 @@
+#include <stdint.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdio.h>
+
+static __inline void datawriteportb(unsigned short _port, unsigned char _data)
+{
+    asm volatile("outb %1, %0"
```

Comment:
I think exposing syscall and call it from here is safer https://man7.org/linux/man-pages/man2/shutdown.2.html